updating/preparing-eus-eus-upgrade.adoc

```diff
@@ -7,20 +7,26 @@ include::_attributes/common-attributes.adoc[]
-You must update from {product-title} 4.10 to 4.11 and then to 4.12. You cannot update from {product-title} 4.10 to 4.12 directly.
-However, beginning with the update from {product-title} 4.10 to 4.11 to 4.12, administrators who wish to update between two Extended Update Support (EUS) versions can do so incurring only a single reboot of non-control plane hosts.
+You must update from {product-title} 4.y to 4.y+1, and then to 4.y+2. You cannot update from {product-title} 4.y to 4.y+2 directly.
+However, administrators who wish to update between two Extended Update Support (EUS) versions can do so incurring only a single reboot of non-control plane hosts.
+
+[IMPORTANT]
+====
+EUS-to-EUS updates are only viable on *even-numbered versions* of {product-title}.
+====
 
 There are a number of caveats to consider when attempting an EUS-to-EUS update.
 
 * EUS-to-EUS updates are only offered after updates between all versions involved have been made available in `stable` channels.
 * If you encounter issues during or after upgrading to the odd-numbered minor version but before upgrading to the next even-numbered version, then remediation of those issues may require that non-control plane hosts complete the update to the odd-numbered version before moving forward.
 * You can do a partial update by updating the worker or custom pool nodes to accommodate the time it takes for maintenance.
 * You can complete the update process during multiple maintenance windows by pausing at intermediate steps. However, plan to complete the entire update within 60 days. This is critical to ensure that normal cluster automation processes are completed including those associated with certificate rotation.
-* Support for RHEL7 workers was removed in {product-title} 4.10 and replaced with RHEL8 workers, therefore EUS-to-EUS updates are not available for clusters with RHEL7 workers. 
+* Support for RHEL7 workers was removed and replaced with RHEL8 workers, therefore EUS-to-EUS updates are not available for clusters with RHEL7 workers. 
 +
-Verify that no RHEL7 workers are present in the 4.10 cluster before performing an EUS-to-EUS upgrade to 4.12. You must replace RHEL7 workers with RHEL8 or RHCOS workers before upgrading to {product-title} 4.12. Red Hat does not support in-place RHEL7 to RHEL8 updates for RHEL workers; those hosts must be replaced with a clean operating system install. 
+Verify that no RHEL7 workers are present in the 4.y cluster before performing an EUS-to-EUS upgrade to 4.y+2. You must replace RHEL7 workers with RHEL8 or RHCOS workers before upgrading to {product-title} 4.y+2. Red Hat does not support in-place RHEL7 to RHEL8 updates for RHEL workers; those hosts must be replaced with a clean operating system install. 
+
+* Until MachineConfigPools are unpaused and the update is complete, features and bugs fixed in the previous version of {product-title} will not be made available.
```

Comment:
Hmm, I don't understand why **previous version** is mentioned. Do you intend to mention "all of the features and bug fixes in the 4.y+2 version  will not be available until MachineConfigPools are unpaused and the update is complete"

Reply:
Removed entirely per @wking !